### PR TITLE
Io project hawls

### PR DESCRIPTION
**Pull request by fpfcap**
i put the hck markdown into the item blog and i wanna have a review first

### CODE REVIEW

File: _posts/2024-3-14 HCK.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="offset-sm-3 col-sm-6 mt-3 mt-md-0">
+        {% include figure.html path="assets/img/20240312HCK .jpg" class="img-fluid rounded z-depth-1" zoomable=true %}
+    </div>
+</div>
+<div class="caption">
+    Figure 1
+</div>
+
+Reference
+Hematopoietic cell kinase (HCK) as a therapeutic target in immune and cancer cells. Oncotarget, Vol. 6, No. 18(2015).
```
Comment (Milokita) [inline _posts/2024-3-14 HCK.md]: the ref format isn;t correct

File: _posts/2024-3-14 HCK.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  Introduction of HCK1 2024/3/12
+date: 2024-3-14 16:00
+description: Introduction of HCK1 2024/3/12
+tags: news images
```
Comment (Milokita) [inline _posts/2024-3-14 HCK.md]: tags not correct

File: _posts/2024-3-14 HCK.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+categories: news
```
Comment (Milokita) [inline _posts/2024-3-14 HCK.md]: not correct

**Review by Milokita — CHANGES_REQUESTED** [review]

File: _posts/2024-3-14 HCK.md (hunk 1 above)
Comment (fpfcap) [reply _posts/2024-3-14 HCK.md]: done

**Review by Milokita — APPROVED** [review]

**Review by Milokita — APPROVED** [review]

**Pull request merged**